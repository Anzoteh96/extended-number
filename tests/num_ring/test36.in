628138022531280376632768999724374929003689382723412894846345417247561683717028811959108793913121008398885140
865266103595716660830891894833598988321170621316021612990004635091475748915947158923124190092604104511508679
519915754980816354570071620860235301638394395966434690385144154927736535699851640536664890902672835223364525
