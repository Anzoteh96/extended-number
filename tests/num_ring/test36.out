Enter the number base b:
Enter two numbers n and m:
b is 628138022531280376632768999724374929003689382723412894846345417247561683717028811959108793913121008398885140
n mod b is 237128081064436284198122895109224059317481238592608718143659217843914065198918346964015396179483096112623539
m mod b is 519915754980816354570071620860235301638394395966434690385144154927736535699851640536664890902672835223364525
n + m is 128905813513972262135425516245084431952186251835630513682457955524088917181741175541571493169034922937102924
n - m is 345350348614900306260820273973363686682776225349586922604860480163739213216095518386459299189931269288144154
n * m is 586966543897325850268319947763431209321289493089759871225572592942852805205213988963693564105733697584874775
n ** m is 398755230043958363124997197857085158626112437810708109419492245870241806556362537130206983902339840557026979
